src/pages/homeCollection/printLabels.js

```diff
@@ -55,17 +55,35 @@ const printLabelsTemplate = (name) => {
 };
 
 const initializeTotalAddressesToPrint = async () => {
-  showAnimation();
-  const totalAddresses = await getTotalAddressesToPrint();
-  const processedAddresses = totalAddresses.data.filter(obj => obj.length !== 0)
-  appState.setState({'totalAddresses': processedAddresses})
-  appState.setState({'totalAddressesLength': processedAddresses.length })
-  hideAnimation();
+  try {
+    showAnimation();
+    const totalAddressCount = await getAddressesToPrintCount();
+    appState.setState({'totalAddressesLength': totalAddressCount.data })
+    hideAnimation();
+  } catch(err) {
+    console.error('Error initializing total addresses to print', err);
+  }
```

Comment:
we can `hideAnimation()` in a finally block. That ensures the app doesn't hang on the loading animation if we hit the catch block.
```suggestion
  try {
    showAnimation();
    const totalAddressCount = await getAddressesToPrintCount();
    appState.setState({'totalAddressesLength': totalAddressCount.data })
  } catch(err) {
    console.error('Error initializing total addresses to print', err);
  } finally {
    hideAnimation();
  }
```